examples/bootstrap-script/README.md

```diff
@@ -0,0 +1,116 @@
+# Bootstrapping your users
+
+Before spawning a notebook to the user, it could be useful to 
+do some preparation work in a bootstrapping process.
+
+One common use case is at follows:
+
+* You are using an LDAPAuthenticator and have no home directory for your user on the host
```

Comment:
*Providing writeable storage for an LDAP user:*

Your JupyterHub is configured to use the LDAPAuthenticator and DockerSpawner.
- The user has no home directory on the host since you are using LDAP.
- When a user has no home directory and DockerSpawner wants to mount a volume,
the spawner will use docker to create a directory.
Since the docker daemon is running as root, the generated directory for the volume mount will not be writeable by the `jovyan` user inside of the container. For the directory to be useful to the user, the permissions on the directory need to be modified for the user to have write access.